src/deps.cljs

```diff
@@ -0,0 +1,7 @@
+{:npm-deps
+ {"karma" "6.3.0"
+  "react" "17.0.2"
```

Comment:
you're using react v17, but v18 is already out, since you are updating react, doesn't it make sense to use latest version?